spark-atlas-connector/src/main/scala/com/hortonworks/spark/atlas/types/external.scala

```diff
@@ -128,17 +132,23 @@ object external {
                        cluster: String,
                        owner: String): Seq[AtlasEntity] = {
     val dbEntity = new AtlasEntity(HIVE_DB_TYPE_STRING)
+    val pathEntity = external.pathToEntity(dbDefinition.locationUri.toString)
```

Comment:
Here I'm assuming location URI would be one of `file://` or `hdfs://`: I'm not familiar with Hive integration and not sure it can be others. 

If the value can be other than FS or HDFS, we have to discard `locationUri` attribute for Hive catalog entities, because `locationUri` now requires FS or HDFS entity (being referenced) instead of normal String (as Hive model defines for `location`).

Reply:
Thank you for update. We usually follow the way Hive SAC works. Hive SAC shows the location ref with this field, doesn't it?

Reply:
There're two kinds of differences: 1) name 2) type. `location` and `locationUri` has both differences. In Spark model, `locationUri` only allows `file://` and `hdfs://`, but able to create 
and link fs/hdfs entity. In Hive model, `location` is just a String, no more.

As we usually follow the way Hive Atlas connector works, then would we want to both renaming attributes and changing type of attributes to fit to Hive model? (as I commented earlier as `the patch can be changed to opposite direction`)